source/speech/manager.py

```diff
@@ -151,6 +151,8 @@ def _reset(self):
 		self._shouldPushWhenDoneSpeaking = False
 
 	def speak(self, speechSequence, priority):
+		if "section" in speechSequence and "First section" in speechSequence:
+			log.error("Found it")
```

Comment:
Looks like this is commited accidentally.

Reply:
Haha, yes I'd say so. Interestingly, this isn't how I found the error.